internal/dbutil/sqliteutil/migrator.go

```diff
@@ -138,9 +138,14 @@ func backupConns(ctx context.Context, sourceDB *sqlite3.SQLiteConn, destDB *sqli
 }
 
 // KeepTables drops all the tables except the specified tables to keep.
-func KeepTables(ctx context.Context, db *sql.DB, tablesToKeep ...string) error {
+func KeepTables(ctx context.Context, db *sql.DB, tablesToKeep ...string) (err error) {
+	txn, err := db.BeginTx(ctx, nil)
```

Comment:
I just noticed that this transaction doesn't get finished in case of errors. @isaachess / @simongui